tests/lang/s01_basics/s00_atoms/t00_builtins.nim

```diff
@@ -0,0 +1,50 @@
+## This test specifies basic operations used in other test.
+
+block builtin_assert:
+  ## `doAssert` command is used throughout the specification to assert
+  ## truhfullness of an expression
+  doAssert true
+
+  doAssert 0 == 0
+
+block built_integer_operations:
+  ## Built-in operations on integer values
+
+  ## Declare integer variable with initial value of `0`
+  var value: int = 0
+
+  ## Assert equality of the variable using `==` operator
+  doAssert value == 0
+
+  ## Increment value of the variale using `inc` command
+  inc value
+
+  doAssert value == 1
+
+  ## Decrement value using `dec` command
+  dec value
+
+  ## Value of the variable returned to 0
+  doAssert value == 0
+
+block assert_type_of_the_expression:
+  ## Nim is a statically typed programming language, which means every
```

Comment:
I'd prefer saying Nimskull over Nim.

Reply:
I think it does not really make sense to mention either of them and "Nim is a statically typed programming language, which means" is also not needed. We just state the fact (specification) - not try to reason about why this made the way it is now.